draft-ietf-mls-protocol.md

```diff
@@ -503,28 +503,16 @@ primitives, defined by the ciphersuite in use:
 
 * A Diffie-Hellman finite-field group or elliptic curve
 * A Key Derivation Function (KDF)
-* A Derive-Public-Key function that produces a public key from a private key
+* A Derive-Key-Pair function that produces an asymmetric keypair
+  from a node secret
 
 A ratchet tree is a left-balanced binary tree, in which each node
 contains up to three values:
 
-* A secret octet string (optional)
+* A path secret octet string (optional)
 * An asymmetric private key (optional)
 * An asymmetric public key
 
-The private key for a node is derived from its secret value using the KDF. The
```

Comment:
There's no reason to delete this para.  It's still true that the key pair from the node derives from the node secret.  You just need to edit it to revert back to what it said before #114, namely that the key pair comes from the node secret via Derive-Key-Pair.